src/vma/dev/cq_mgr.cpp

```diff
@@ -1347,21 +1347,15 @@ cq_mgr* get_cq_mgr_from_cq_event(struct ibv_comp_channel* p_cq_channel)
 
 void cq_mgr::modify_cq_moderation(uint32_t period, uint32_t count)
 {
-#ifdef DEFINED_IBV_EXP_CQ_MODERATION
-	struct ibv_exp_cq_attr cq_attr;
+	vma_ibv_cq_attr cq_attr;
```

Comment:
How this code works in case neither IBV_EXP_CQ_MODERATION nor IBV_CQ_MODERATION is defined?

Reply:
it does not :)